modules/subscriber/cloudwatch.tf

```diff
@@ -0,0 +1,13 @@
+resource "aws_cloudwatch_log_group" "subscriber_log_group" {
+  name              = "/aws/lambda/${var.name}"
+  retention_in_days = 365
+}
+
+resource "aws_cloudwatch_log_subscription_filter" "test_lambdafunction_logfilter" {
```

Comment:
Is this leftover from testing?

Reply:
this was actually the one resource that I had to add that didn't seem like it was present in the SAM app. otherwise i didnt get any logs flowing from lambda => kinesis.

Reply:
Right, you're hardcoding the subscription. If you want it to be discovered you'd have to ensure that the subscriber is allowed to pick up `/aws/lambda/${var.name}`. You can do that in the test by setting `log_group_name_patterns = ["${var.name}"]` or equivalent.

Reply:
Not by default. A default install should set up the machinery to collect logs. You then have one of two paths:
- you can create static subscription filters, like you did here. This is the more appealing solution if you have everything in IaC, because you don't need extra infrastructure. It gets really burdensome if you have hundreds of log groups you want to subscribe to.
- you can dynamically subscribe using our `subscriber` module.

Reply:
should I output the destination iam policy arn then as well? and include an example in the readme of how to create a subscription

Reply:
I think that is useful but you can add that in a separate commit as a fix later if necessary. No need to get it all right in first go.